qt/applications/workbench/workbench/plugins/workspacewidget.py

```diff
@@ -93,6 +96,24 @@ def _do_plot_spectrum(self, names, errors, overplot):
 
         plot_from_names(names, errors, overplot)
 
+    def _do_plot_bin(self, names, errors, overplot):
```

Comment:
There are unit tests covering this file. Can they be updated to add a test for this scenario?